README.md

```diff
@@ -31,7 +31,7 @@ LPH.
 
  - 📂 Backend documentation's directory is [here](doc/backend) 
  - [📝 Backend Documentation](server/README.md)
- - [📝 API Documentation](doc/backend/API-documentation.md)
+ - [📝 API Documentation](server/README.md#how-to-play)
```

Comment:
These changes shouldn't be in this PR they are already inlcuded in the main branch see : c657a4e.